content/en/blog/2021-12-23-data-curation-baleen.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Data Curation: A Whale of a Problem"
+slug: "data-curation-baleen"
+date: 2021-12-24T11:37:43-05:00
+draft: false
+image_webp: images/blog/whale.webp
+image: images/blog/whale.jpg
+author: Nabiha Naqvie
+description: "Recent developments in Artificial Intelligence (AI) and Natural Language Processing (NLP) indicate a growing need for better mechanisms for collecting curated datasets, and introduces a new open source tool for this purpose."
+---
+
+Once you become an AI/ML practitioner, you quickly realize that the machine learning work is often the *least* challenging step in the pipeline. So what's still really, really hard? Getting good data! In this post, we'll explore why that is and introduce a new open source tool for data curation, Baleen. <!--more-->
+
+The world of AI has seen a rapid increase in its practical usage, as AI companies and startups raised $33 billion during 2020.[^1] Many companies like Urbint are using AI and real-world data to predict and prevent incidents that threaten critical infrastructure, workers, the community, and the environment.[^2] Similarly, DataRobot claims to have the only AI Cloud platform with an AI-native strategic team to help its customers use and understand their data.[^3] Natural language processing applications have also seen a surge; 60% of technology leaders report that their NLP budgets grew by at least 10% in 2021, while 33% reported a 30% increase, and 15% reported they had more than doubled.[^4]
+
+Although this accelerated movement towards AI, NLP, and Machine Learning has added value to how we interact with and perceive the world, we are consistently reminded to be cautious about the acceleration of technology. The fears of some detractors are difficult to take seriously, but even some of the most brilliant minds have expressed concerns:
+
+> We cannot quite know what will happen if a machine exceeds our own intelligence, so we can't know if we'll be infinitely helped by it, or ignored by it and sidelined, or conceivably destroyed by it. -Stephen Hawking [^5]
```

Comment:
I wasn't sure which Stephen Hawking quote you were thinking of so I just threw this one in as a placeholder -- please update it and the footnote link if you have another one in mind!